commands/fun/anime.js

```diff
@@ -24,18 +24,18 @@ class Anime extends Command {
       return ["1", "2", "3", "4", "5"].includes(msg.content);
     }
     if (args.length < 1) return message.reply("You must add an anime to search for");
-    let msg = await message.channel.send("*fetching information from kitsu!*");
+    let msg = await message.channel.send("Fetching information from Kitsu!");
     try {
       const { data } = await kitsu.fetch("anime", { filter: { text: args.join("-") } });
       if (data.length < 1) throw new this.client.methods.errors.AnimeError("No result found");
-      msg = await msg.edit(`Okay i found 5 possible matches which do you want to see? (just write the first number, it will be canceled after 60 seconds)${this.makeTitles(data)}`);
+      msg = await msg.edit(`Okay, I found 5 possible matches!~ Which one do you want to see~\n(Just write the number, the one you want to see!)${this.makeTitles(data)}`);
```

Comment:
Missing timer notice, this is required else the user will be confused on when it deletes itself.